config/routes.rb

```diff
@@ -6,7 +6,7 @@
   get '/github/callback' => 'github#callback'
 
   resources :feedback, only: %i[index show new create]
-  resources :log, except: %i[edit new show]
+  resources :log_entries, only: %i[index create update destroy], path: '/log'
```

Comment:
```suggestion
  resources :log_entries, only: %i[index create update destroy], path: 'log'
```

(At least according to the example in the Rails guide; seems to work the same either way)